web/appsettings.json

```diff
@@ -1,21 +1,28 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Warning",
-      "System": "Warning"
+    "Logging": {
+        "LogLevel": {
+            "Default": "Warning",
+            "System": "Warning"
+        }
+    },
+    "LoggingPath": "C:\\nlog", //TODO - change to production logging location,
+    "AllowedHosts": "*",
+    "IPAddressAllowlistConfiguration": {
+        "InternalAllowlist": [
+            "::1", // IPv6 localhost
+            "127.0.0.1", // IPv4 localhost
+            "192.168.32.150" // Dan
+        ],
+        "ExternalAllowlist": []
+    },
+    "Cas": {
+        "CasBaseUrl": "https://cas.ucdavis.edu/cas/"
+    },
+    "EmailNotifications": {
+        "PrimaryEvaluatorRemoved": {
+            "To": ["lgissvoran@ucdavis.edu"],
```

Comment:
@bsedwards, I am unsure if it is okay to put the email address in here on a public repo. Is there a way to inject the config into the Jenkins build?